src/config/firefox-desktop.js

```diff
@@ -103,13 +103,34 @@ export default {
       ...this.dimensions,
     });
   },
+  snakeCase(line) {
+    // This is a direct translation from bigquery-etl's python snake_case transformation,
+    // so probe names queried from Glam's DB are snake_cased the same as in the ETL.
+    // This is only required for scalar probes.
+    // source: https://github.com/mozilla/bigquery-etl/blob/533ec4b203716c60ca10a62c6d9c302926b052ae/bigquery_etl/util/common.py#L27
+    const pattern =
+      /\b|(?<=[a-z][A-Z])(?=\d*[A-Z])|(?<=[a-z][A-Z])(?=\d*[a-z])|(?<=[A-Z])(?=\d*[a-z])/;
+    const reg = /[^\w]|_\./;
+    function reverse(s) {
+      return s.split('').reverse().join('');
+    }
+    const subbed = reverse(line).replace(reg, ' ');
+    const words = subbed.split(pattern).filter((x) => x.trim());
+    return reverse(words.join('_').toLowerCase());
+  },
```

Comment:
I wonder if we should move this method to `glam/src/utils` and make this its own JS module so we can reuse it elsewhere if needed?

Reply:
I think that's a good idea. I'll accept it

Reply:
I suggest writing a test for this method too if you are going to make it a module.

Reply:
nvm I'll take your other suggestion and use lodash's snakeCase(). Thanks!